index.html

```diff
@@ -35,6 +36,99 @@ <h2>
+            
+            <img src="Images/GymFit.png" alt="GymFit">
+            <h2 id="Multi">Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="boxes white">
+                <div>css</div>
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+            <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+        <article class="repeated">
+            <h2>Profesional Art Printing Data</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="boxes black">
+                <div>html</div>
+                <div>bootstrap</div>
+                <div>Ruby</div>
+            </div>
+                <div class="Project">See Project</div>
+        </article>
+    </div>
+    <div class="aboutme">
```

Comment:
In our requirements for the project, the about me section should be wrapped in a section tag rather than a div. Kindly fix it to meet the expected requirements 😁